data-service/src/main/java/uk/gov/gchq/palisade/service/data/exception/SerialiserConstructorNotFoundException.java

```diff
@@ -20,30 +20,11 @@
  * A {@link RuntimeException} that is thrown when incorrect values are used for the service Prepopulation.
  */
 public class SerialiserConstructorNotFoundException extends RuntimeException {
-
-    /**
-     * Constructs a new {@link SerialiserConstructorNotFoundException} with the cause.
-     *
-     * @param cause     a {@link Throwable} that caused the error
-     */
-    public SerialiserConstructorNotFoundException(final Throwable cause) {
-        super(cause);
-    }
-
-    /**
-     * Constructs a new {@link SerialiserConstructorNotFoundException} with the specified detail message.
-     *
-     * @param message   a {@link String} value detailing the error
-     */
-    public SerialiserConstructorNotFoundException(final String message) {
-        super(message);
-    }
```

Comment:
Is it not worth keeping these constructors in case they're needed in the future?

Reply:
Unused, and we really should be using only the variant with both a throwable and a message

Reply:
That's fair enough